packages/client/src/components/image/Image.tsx

```diff
@@ -1,13 +1,10 @@
 import { ImageBody, ImageWrapper } from './Image.styled';
 
-export interface ImageProps {
-  src: string;
-  alt: string;
-  height?: string;
-  width?: string;
+export interface ImgProps extends React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement> {
```

Comment:
That's it, what I also do as an improvement `React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement>` to be in the global scope `client/types` folder  where we have `index.ts` as source export of global(reusable) types and for this particular example have `image.ts` file where we put React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement> and give appropriate name e.g. `TImage`